code/modules/mob/living/carbon/brain/life.dm

```diff
@@ -31,8 +31,13 @@
 /mob/living/carbon/brain/Life()
 	. = ..()
 	if(.)
-		if(!container && (health < HEALTH_THRESHOLD_DEAD && check_death_method() || ((world.time - timeofhostdeath) > config.revival_brain_life)))
+		if((world.time - timeofhostdeath) > config.revival_brain_life)
 			death()
 
+/mob/living/carbon/brain/update_stat(reason = "none given")
+	if(!container && (health < HEALTH_THRESHOLD_DEAD && check_death_method()))
```

Comment:
Wait I think I might have totally messed up the logic here
health being below the threshold should be grounds for death regardless of containerness
meanwhile the time thing would depend on the container being present or not - as-is, the brain is going to die whether in the MMI or not